src/scanner.cc

```diff
@@ -599,6 +599,9 @@ bool uninitialized(State & state) { return !indent_exists(state); }
 Condition column(uint32_t col) {
   return [=](State & state) { return state::column(state) == col; };
 }
+static bool column_v2(uint32_t col, State &state) {
+  return state::column(state) == col;
```

Comment:
:+1:

Reply:
I think you're looking at intermediate commits?

Reply:
Yes I was. Disregard outdated comments. But this was a nice change nonetheless.